templates/ez_segmenter.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>Let's clear all output from this cell, no need to leave this in.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/958/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Done</p>